monitor.py

```diff
@@ -529,7 +529,13 @@ def parse_entry(self, line: bytes) -> MutableMapping[str, Any]:  # noqa: C901, C
                 # Odyssey: bool
                 self.cmdr = entry['Commander']
                 # 'Open', 'Solo', 'Group', or None for CQC (and Training - but no LoadGame event)
-                self.mode = entry.get('GameMode')
+                if not entry.get('Ship') and not entry.get('GameMode') or entry.get('GameMode').lower() == 'CQC':
+                    logger.trace(f'loadgame to cqc: {entry}')
```

Comment:
All new logging.trace should probably make use of a --trace-on switch, so it can be toggled on/off as needed.

See the last part of https://github.com/EDCD/EDMarketConnector/blob/main/Contributing.md#use-logging-not-print-and-definitely-not-sysstdoutwrite

Reply:
I'll add it, no problems, but for some unknown reason, `trace_on` in this place always is an empty list so it doesn't gonna working.